content/documentation/4-batch-developer-guides/2-batch/2-spring-batch.md

```diff
@@ -8,34 +8,310 @@ description: 'Create a Spring Batch Job'
+		assertThat(billStatement.getId()).isEqualTo(1);
+		assertThat(billStatement.getMinutes()).isEqualTo(500);
+		assertThat(billStatement.getDataUsage()).isEqualTo(1000);
+
+	}
+
+	public static class BillStatement extends Usage {
+
+		public BillStatement(Long id, String firstName, String lastName, Long minutes, Long dataUsage, double billAmount) {
+			super(id, firstName, lastName, minutes, dataUsage);
+			this.billAmount = billAmount;
+		}
+
+		private double billAmount;
+
+		public double getBillAmount() {
+			return billAmount;
+		}
+
+		public void setBillAmount(double billAmount) {
+			this.billAmount = billAmount;
+		}
+	}
+
+}
+
+```
 
 ## Deployment
 
 Deploy to local, Cloud Foundry and Kubernetes
 
 ### Local
 
-get the jar run it bla
+1.  Now let’s take the next step of building the project.
+    From a command line change directory to the location of your project and build the project using maven:
+    `mvn clean package`.
+
+2.  Now let’s execute the application with the configurations required to process the usage information in the database.
+
+    To configure the execution of the billrun application utilize the following arguments:
+
+    1. _spring.datasource.url_ - set the URL to your database instance. In the sample below we are connecting to a mysql `task` database on our local machine at port 3306.
+    1. _spring.datasource.username_ - the user name to be used for the MySql database. In the sample below it is `root`
+    1. _spring.datasource.password_ - the password to be used for the MySql database. In the sample below it is `password`
+    1. _spring.datasource.driverClassName_ - The driver to use to connect to the MySql database. In the sample below it is `com.mysql.jdbc.Driver'
+    1. _spring.datasource.initialization-mode_ - initializes the database with the BILL_STATEMENTS and BILL_USAGE tables required for this app. In the sample below we state that we `always` want to do this. This will not overwrite the tables if they already exist.
+    1. _spring.batch.initialize-schema_ - initializes the database with the tables required for Spring Batch. In the sample below we state that we `always` want to do this. This will not overwrite the tables if they already exist.
+
+    ```bash
+    $ java -jar target/billrun-1.0.0.BUILD-SNAPSHOT.jar \
+    --spring.datasource.url=jdbc:mysql://localhost:3306/task?useSSL=false \
+    --spring.datasource.username=root \
+    --spring.datasource.password=password \
+    --spring.datasource.driverClassName=com.mysql.jdbc.Driver \
+    --spring.datasource.initialization-mode=always \
+    --spring.batch.initialize-schema=always
+    ```
+
+3.  Log in to the `mysql` container to query the `BILL_STATEMENTS` table.
+    Get the name of the `mysql`pod using`kubectl get pods`, as shown above.
+    Then login:
+
+<!-- Rolling my own to disable erroneous formating -->
+<div class="gatsby-highlight" data-language="bash">
+<pre class="language-bash"><code>$ docker exec -it mysql bash -l
```

Comment:
In your earlier docker command, you named the container `some-mysql` instead of just `mysql`.

Reply:
Fixed the docker command above.